TradeItIosTicketSDK2/CloseableViewController.swift

```diff
@@ -16,9 +16,9 @@ class CloseableViewController: UIViewController {
             self.createCloseButton()
         }
 
-        TradeItThemeConfigurator.configureBarButtonItem(button: self.navigationItem.leftBarButtonItem)
-        TradeItThemeConfigurator.configureBarButtonItem(button: self.navigationItem.rightBarButtonItem)
-        TradeItThemeConfigurator.configureBarButtonItem(button: self.navigationItem.backBarButtonItem)
+        //TradeItThemeConfigurator.configureBarButtonItem(button: self.navigationItem.leftBarButtonItem)
+        //TradeItThemeConfigurator.configureBarButtonItem(button: self.navigationItem.rightBarButtonItem)
+        //TradeItThemeConfigurator.configureBarButtonItem(button: self.navigationItem.backBarButtonItem)
```

Comment:
These need to be moved. Yahoo uses `CloseableViewController` but not the themes.